app/crud.py

```diff
@@ -0,0 +1,23 @@
+from sqlalchemy.orm import Session
+from . import models, schemas
+
+def create_city(db: Session, city: schemas.CityCreate):
+    db_city = models.City(name=city.name, additional_info=city.additional_info)
+    db.add(db_city)
+    db.commit()
+    db.refresh(db_city)
+    return db_city
+
+def get_city(db: Session, city_id: int):
+    return db.query(models.City).filter(models.City.id == city_id).first()
+
+def get_cities(db: Session, skip: int = 0, limit: int = 10):
+    return db.query(models.City).offset(skip).limit(limit).all()
+
+def delete_city(db: Session, city_id: int):
```

Comment:
Similar to the `create_city` function, consider adding error handling for potential exceptions during the deletion process, such as when the city does not exist or there are database connection issues.